d3rlpy/base.py

```diff
@@ -480,7 +487,9 @@ def fitter(
             shuffle: flag to shuffle transitions on each epoch.
             callback: callable function that takes ``(algo, epoch, total_step)``
                 , which is called every step.
-
+            epoch_callback: callable function that takes 
+                ``(algo, epoch, total_step)``, which is called every epoch.
```

Comment:
I'd like to explicitly explain it's called at the end of epochs.
```
``(algo, epoch, total_step)``, which is called at the end of every epoch.
```

Reply:
Makes sense - have included